#BodySizeArray.msg

Header header
BodySize[] distances
